Lecture4.tex

```diff
@@ -250,7 +250,7 @@ \subsection{Infinite Horizon Problems}
 \[J_\alpha ^\pi (s) = {E^\pi }\left(\sum\limits_{t = 0}^\infty  {{\alpha ^t}r({s_t},{a_t})} |{s_0} = s\right) \equiv {E^{\pi ,s}}\left(\sum\limits_{t = 0}^\infty  {{\alpha ^t}r({s_t},{a_t})} \right)\]
 Here $0 < \alpha  < 1$ is the discount factor. Mathematically, the discount factor ensures convergence of the sum (whenever the reward sequence is bounded). This make the problem "well behaved", and relatively easy to analyze.
 \paragraph{Average return:}  Here we are interested in maximizing the long-term average return. The most common definition of the long-term average return is
```

Comment:
doesn't it mean that when T -> infinity we want to maximize the average return? thus, Sup{E[...]}